sklearn/ensemble/bagging.py

```diff
@@ -289,6 +293,9 @@ def _fit(self, X, y, max_samples, sample_weight=None):
         # Check parameters
         self._validate_estimator()
 
+        if max_depth is not None:
+            self.base_estimator_.max_depth = max_depth
+
         # if max_samples is float:
         if not isinstance(max_samples, (numbers.Integral, np.integer)):
             max_samples = int(self.max_samples * X.shape[0])
```

Comment:
This is another left-out bug. It should be `max_samples = int(max_samples * X.shape[0])` (without `self`).